test/reputation-mining/happy-paths.js

```diff
@@ -72,15 +74,21 @@ contract("Reputation Mining - happy paths", accounts => {
     colonyNetwork = await setupColonyNetwork();
     ({ metaColony, clnyToken } = await setupMetaColonyWithLockedCLNYToken(colonyNetwork));
 
+    // Replace addDomain with the addDomain implementation with no restrictions on depth of subdomains
+    const noLimitSubdomains = await NoLimitSubdomains.new();
+    const resolverAddress = await colonyNetwork.getColonyVersionResolver(1);
```

Comment:
I don't think I understand how this would break, or how `getCurrentColonyVersion` would help solve the problem (though maybe that becomes obvious once I understand the problem)

Reply:
`getColonyVersionResolver` returns the Resolver instance of Colony contract version 1 in this instance.
However, any new colonies we create, will always be using the latest Colony contract version, which currently is also 1 but will obviously be incremented over time.

Reply:
I'm going to have to blame being distracted by Brexit stuff for why I was such an idiot here.